github_connector/demo/github_analysis_rule_demo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+Copyright 2020 Tecnativa - Víctor Martínez
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <record id="github_analysis_rule_1_demo" model="github.analysis.rule">
+        <field name="name">Python files</field>
+        <field name="group_id" ref="github_analysis_rule_group_1_demo" />
+        <field name="paths">*.py</field>
```

Comment:
Isn't this excluding Python files in tests? (the same for JS tests).

Reply:
This code analyze all .py files (tests included)